lib/exceptions/BadRequestException.ts

```diff
@@ -3,7 +3,7 @@ import { HttpException } from './HttpException';
 export class BadRequestException extends HttpException {
   public name = 'BadRequestException';
 
-  public constructor(message?: string, errors?: string[]) {
+  public constructor(message: string = 'Bad request', errors?: string[]) {
```

Comment:
I believe default messages should reflect the web standards, thus `Bad Request` would be the correct message here.